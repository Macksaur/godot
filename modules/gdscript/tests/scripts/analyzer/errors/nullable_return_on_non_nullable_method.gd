func test():
    non_nullable_return()

func non_nullable_return() -> Vector2:
    return nullable_return() # Should error

func nullable_return() -> Vector2?:
    return null