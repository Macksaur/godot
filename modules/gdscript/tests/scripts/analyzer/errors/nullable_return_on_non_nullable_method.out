GDTEST_ANALYZER_ERROR
Cannot return nullable value of type "Vector2?" because the function return type is "Vector2".
